Samples/Adventure/AdventureGrainInterfaces/AdventureGrainInterfaces.csproj

```diff
@@ -1,85 +1,15 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{D0506AE9-D2C7-48AC-8ABC-4A5A74EE56C2}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>AdventureGrainInterfaces</RootNamespace>
+    <TargetFramework>net461</TargetFramework>
     <AssemblyName>AdventureGrainInterfaces</AssemblyName>
-    <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
-    <TargetFrameworkProfile />
+    <PackageId>AdventureGrainInterfaces</PackageId>
+    <NetStandardImplicitPackageVersion>1.6.0</NetStandardImplicitPackageVersion>
```

Comment:
what is this line trying to set ? why isn't the version 2.0.0?

Reply:
VS added this

Reply:
Can't we explicitly target .NET Core 2.0?

Reply:
This sample target the full framework 4.6.1, to shot that users can still use it/ Is it an issue?